beetsplug/badfiles.py

```diff
@@ -97,14 +102,25 @@ def check_bad(self, lib, opts, args):
             ext = os.path.splitext(item.path)[1][1:].decode('utf8', 'ignore')
             checker = self.get_checker(ext)
             if not checker:
-                self._log.debug(u"no checker available for {}", ext)
+                self._log.error(u"no checker specified in the config for {}",
+                                ext)
                 continue
             path = item.path
             if not isinstance(path, six.text_type):
                 path = item.path.decode(sys.getfilesystemencoding())
-            status, errors, output = checker(path)
+            try:
+                status, errors, output = checker(path)
+            except CheckerCommandException as e:
+                if e.errno == errno.ENOENT:
+                    self._log.error(
+                            u"command not found: {} when validating file: {}",
+                            e.checker,
+                            e.path)
```

Comment:
I think Travis is complaining about the indentation here. The indented lines should just be one "tab" (4 spaces) indented, and the closing parenthesis should be on a line by itself (not indented).